lib/wsdl.js

```diff
@@ -1112,6 +1112,12 @@ WSDL.prototype._initializeOptions = function (options) {
   } else {
     this.options.returnFault = false;
   }
+  if (options.returnNilAsNull !== undefined) {
```

Comment:
@herom I went through a few different iterations of naming (convertNilToNull, omitNullValues, etc). I thought that returnNilAsNull was consistent with the option directly above in the code returnFault.  Glad to change to whatever maintainers feel is consistent with project.